core/src/main/java/org/fao/geonet/kernel/Thesaurus.java

```diff
@@ -965,18 +983,29 @@ private void retrieveThesaurusInformation(Path thesaurusFile, String defaultTitl
                 this.defaultNamespace = DEFAULT_THESAURUS_NAMESPACE;
             }
 
-            Element dateEl = Xml.selectElement(thesaurusEl, "skos:ConceptScheme/dcterms:issued|skos:Collection/dc:date", theNSs);
+            Element issuedDateEl = Xml.selectElement(thesaurusEl, "skos:ConceptScheme/dcterms:issued|skos:Collection/dc:date", theNSs);
+            DateFormat dateFormatYearOnly = new SimpleDateFormat("yyyy");
```

Comment:
Why is the format YYYY? It could be in many different formats we don't know what format it will be in.
Why are we  checking any format?  Why not simply take the value from dcterms:issued exacly as entered as a text string?

Reply:
I have the new code to use the text as is.